### ISSUE TITLE
cannot activate active backup for business by alpha9

### ISSUE THREAD

**Issue by Pascalleung** [fbelavenuto/arpl#183]
DSM 7.1.1- 42692 ( Get activation status with serial )
i had update IP: port and serial no.

http://192.168.81.113:5000/webapi/entry.cgi?api=SYNO.ActiveBackup.Activation&method=set&version=1&activated=true&serial_number=

{"error":{"code":119},"success":false}

alpha6 can activate. thanks.

**Comment by fbelavenuto** [comment]
I don't really know this service. A valid serial/mac is required, right?

**Comment by Pascalleung** [comment]
yes it need serial ; but random generate was accept. alpha6 was ok. but alpha 7,8,9 will error 119.

**Comment by Cryental** [comment]
For me it works with real S/N and mac address. Maybe you can try to get them from the marketplace and try again?

**Comment by Cryental** [comment]
@Pascalleung solved with last release?
